tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1382,19 +1382,21 @@ class TftrtAlgorithmSelector : public nvinfer1::IAlgorithmSelector {
       nvinfer1::DataType datatype =
           algoChoices[i]->getAlgorithmIOInfo(0).getDataType();
 
-      if (implementation == static_cast<int64_t>(LayerImpl::kSHUFFLE) &&
-          (
-#if IS_TRT_VERSION_GE(8, 0, 0, 0)
-              // Reject shuffle node when input format is linear row major INT8
-              // format
-              format == nvinfer1::TensorFormat::kLINEAR &&
-              datatype == nvinfer1::DataType::kINT8
-#else
-              // Reject shuffle node when input format is 32-wide channel
-              // vectorized row major FP32 format
-              format == nvinfer1::TensorFormat::kCHW32
+      bool reject_tactic = false;
+#if IS_TRT_VERSION_GE(8, 0, 0, 0) && !IS_TRT_VERSION_GE(8, 0, 3, 0)
```

Comment:
We don't have any test for this, right?
How do you test the change? Would it be easy to add tests for that? 

Please also squash the commits into one when you push next time.

Reply:
We added these rejections earlier in https://github.com/tensorflow/tensorflow/pull/50408 due to internal-to-NV test failures seen in TRT 7.x and 8.0, respectively.  This PR clarifies the readability of the logic and fixes the conditionals so that the TRT 8.0 workaround does not apply to TRT 8.2, which fixed that bug.